service/src/main/java/uk/gov/hmcts/reform/fpl/service/RepresentativeService.java

```diff
@@ -211,4 +214,31 @@ private void associatedRepresentativeWithParty(CaseData caseData, Element<Repres
                 return Optional.empty();
         }
     }
+
+    public List<Element<Representative>> getRepresentativePartiesToNotify(CallbackRequest request) {
+        CaseData caseData = mapper.convertValue(request.getCaseDetails().getData(), CaseData.class);
+        CaseData caseDataBefore = mapper.convertValue(request.getCaseDetailsBefore().getData(), CaseData.class);
+
+        if (isNotEmpty(caseDataBefore.getRepresentatives())) {
+            if (!caseDataBefore.getRepresentatives().containsAll(caseData.getRepresentatives())) {
+                return getChangedRepresentatives(caseData,caseDataBefore);
+            }
+
+        } else {
+            if (!caseData.getRepresentatives().isEmpty()) {
+                return caseData.getRepresentatives();
+            }
+        }
+
+        return null;
```

Comment:
Could return an empty list here to reduce the need for null checks

Reply:
done